### ISSUE TITLE
Tooling - Beta8 Project Templates need a Bower.json change to stop VSO Build failures

### ISSUE THREAD

**Issue by jderus** [aspnet/Home#1073]
FYI for those currently using VSO Build and Beta8 Tools.
(Tooling team might consider updating to more recent bower packages.)

Current ASP5 Project Templates have this in bower.json

```
    "jquery-validation": "1.11.1",
    "jquery-validation-unobtrusive": "3.2.2"
```

When this gets into a simple VSO Build it causes build breaks as follows:
invalid-meta jquery-validation-unobtrusive is missing "ignore" entry in bower.json
invalid-meta jquery-validation is missing "main" entry in bower.json

Seems like this has some info on potential root cause: https://github.com/bower/bower/issues/1388 

In the meantime for anyone using the current Beta8 tools ... to fix this, just update the offending packages. I can confirm these work to unblock your VSO Build:

```
    "jquery-validation": "1.14.0",
    "jquery-validation-unobtrusive": "3.2.4"
```

**Comment by Eilon** [comment]
@madskristensen @DamianEdwards @sayedihashimi ?

**Comment by Eilon** [comment]
Or even @rustd

**Comment by sayedihashimi** [comment]
That should be @rustd to make the change.

**Comment by aspnet-hello** [comment]
This issue is being closed because it has not been updated in 3 months.

We apologize if this causes any inconvenience. We ask that if you are still encountering this issue, please log a new issue with updated information and we will investigate.
